lib/kitchen/provisioner/dokken.rb

```diff
@@ -128,6 +130,12 @@ def cleanup_dokken_sandbox
         debug("Cleaning up local sandbox in #{sandbox_path}")
         FileUtils.rmtree(Dir.glob("#{sandbox_path}/*"))
       end
+
+      def chef_executable
+        return "HAB_LICENSE='accept-no-persist' #{config[:hab_chef_binary]} pkg exec ashiqueps/chef-infra-client -- chef-client " if instance.driver.installer == "habitat"
```

Comment:
hab pkg location needs to be removed from ashiqueps

Reply:
Done.